src/pull_request.ts

```diff
@@ -278,3 +282,25 @@ async function submitReview(
     );
   }
 }
+
+function shouldIgnorePullRequest(pull_request: { body?: string }) {
```

Comment:
Add JSDoc documentation to describe the function's purpose, parameters, and return value. This will improve code maintainability and help other developers understand the function's behavior:
```typescript
/**
 * Checks if the pull request should be ignored based on special commands in its description.
 * @param pull_request - The pull request object containing the description
 * @param pull_request.body - The description text of the pull request
 * @returns {boolean} True if the PR should be ignored, false otherwise
 */
function shouldIgnorePullRequest(pull_request: { body?: string }) {
```